src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -2812,6 +2813,66 @@ def test_sql_managed_instance_mgmt(self):
                  expect_failure=True)
 
 
+class SqlManagedInstancePoolScenarioTest(ScenarioTest):
+    @record_only()
```

Comment:
May I know why you want to make it record_only() for your test?

Reply:
Instance Pool's operations are time-consuming and resource intensive. Maybe it would make sense for us to have it as record_only rather than making live calls everytime?

Reply:
Can be re-recorded. Will take a very long time though.